src/_includes/page-github-links.html

```diff
@@ -4,7 +4,7 @@
 {% endcomment -%}
 
 {% assign repo = repo | default: site.repo.this -%}
-{% capture path -%} {{repo}}/tree/{{site.branch}}/{{page.inputPath}} {%- endcapture -%}
+{% capture path -%} {{repo}}/tree/{{site.branch}}/{{page.inputPath | regex_replace: '^./'}} {%- endcapture -%}
```

Comment:
Would the built-in Liquid replace function work here?

```suggestion
{% capture path -%} {{repo}}/tree/{{site.branch}}/{{page.inputPath | replace: './', ''}} {%- endcapture -%}
```

Reply:
It's working. 😄